moto/logs/responses.py

```diff
@@ -7,6 +9,9 @@
 
 
 class LogsResponse(BaseResponse):
+    def __init__(self):
+        super().__init__()
```

Comment:
Travis isn't too happy about this - it fails in Python 2. Any particular reason you added it?

Reply:
I believe I was using it before re-factoring. Shouldn't be needed anymore.